utils/oscap-remediate-offline-service

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+
+# SPDX-License-Identifier: GPL-2.0-or-later
+# Copyright (C) 2021 Red Hat Inc., Durham, North Carolina.
+#
+# Evgenii Kolesnikov <ekolesni@redhat.com>
+#
+# The offline system remediation (systemd's /system-update) service wrapper
+
+log() {
+    echo "${1}" | logger -t oscap-remediate-offline -p ${2:-info}
+}
+
+config=$(readlink -f /system-update)
+
+if [[ "$(basename ${config})" != "oscap-remediate-offline.conf.sh" ]]; then
+    log "The /system-update symlink was not created by oscap-remediate-offline, ignoring"
+    systemd-notify --ready --status="Skipped"
+    exit 0
+fi
+
+rm -f /system-update
+
+log "Found config: ${config}"
+
+source "$config"
+
```

Comment:
What if the config file is broken and doesn't contain OSCAP_REMEDIATE_DS or OSCAP_REMEDIATE_PROFILE_ID ?

Reply:
I've put all these checks in `oscap-remediate-offline` in the beginning. But then recently I realized that it is inherently not the only way to prime the service (think Image Builder) and we have to re-check it in the service itself. Hence `TODO: Check` entries there.

Reply:
See: https://github.com/OpenSCAP/openscap/pull/1838/files#diff-ca5a0596040ff7754e2ce486cfab14fbd60d1b804d5a7dcbf9649529031fac49R52

Reply:
Should be fixed now